Examples/Physics_applications/spacecraft_charging/inputs_test_rz_spacecraft_charging_picmi.py

```diff
@@ -49,14 +53,15 @@ def correct_space_charge_fields(self, q=None):
             q = compute_actual_charge_on_spacecraft()
 
         # Correct fields so as to recover the actual charge
-        Er = ExWrapper(include_ghosts=True)
-        Er[...] += (q - q_v) * self.normalized_Er
-        Ez = EzWrapper(include_ghosts=True)
-        Ez[...] += (q - q_v) * self.normalized_Ez
-        phi = PhiFPWrapper(include_ghosts=True)
-        phi[...] += (q - q_v) * self.normalized_phi
+        Er = warpx.multifab("Efield_fp", dir=self.dir_r, level=0)
+        Er.saxpy(Er, (q - q_v), self.normalized_Er, 0, 0, 1, 0)
+        Ez = warpx.multifab("Efield_fp", dir=self.dir_z, level=0)
+        Ez.saxpy(Ez, (q - q_v), self.normalized_Ez, 0, 0, 1, 0)
+        phi = warpx.multifab("phi_fp", level=0)
+        phi.saxpy(phi, (q - q_v), self.normalized_phi, 0, 0, 1, 0)
```

Comment:
Note: #5613 will change `saxpy`
```suggestion
        Er = warpx.multifab("Efield_fp", dir=self.dir_r, level=0)
        Er.saxpy(q - q_v, self.normalized_Er, 0, 0, 1, 0)
        Ez = warpx.multifab("Efield_fp", dir=self.dir_z, level=0)
        Ez.saxpy(q - q_v, self.normalized_Ez, 0, 0, 1, 0)
        phi = warpx.multifab("phi_fp", level=0)
        phi.saxpy(q - q_v, self.normalized_phi, 0, 0, 1, 0)
```